docs/conf.py

```diff
@@ -554,3 +554,4 @@ def setup(app):
     # # Ignore .ipynb files
     app.registry.source_suffix.pop(".ipynb", None)
     app.registry.source_suffix.pop(".py", None)
+    app.registry.source_suffix.pop(".py.md5", None)
```

Comment:
ok, but I think all this has no effect on the sphinx gallery warning unfortunately

Reply:
We can let it like this for the moment